apps/pano/app/routes/posts/$id.edit.tsx

```diff
@@ -31,6 +31,12 @@ type ActionData = {
     message: string;
   };
 };
+const PostSchema = z.object({
+  url: z.string().url("Lütfen geçerli bir URL adresi girin."),
+  title: z.string().min(2, "Başlık en az iki harfli olmalıdır."),
+  content: z.string().min(1, "İçerik en az 1 harften oluşmalıdır."),
```

Comment:
these changes require users to fill in every input field for sending a post.
for this one, there are three things to consider:
1- user can send a post with `url` and `title`
2- user can send a post with `title` and `content`
3- user can send a post with `url`, `title` and `content`